xbmc/ServiceManager.cpp

```diff
@@ -174,6 +176,11 @@ ADDON::CBinaryAddonManager &CServiceManager::GetBinaryAddonManager()
   return *m_binaryAddonManager.get();
 }
 
+ADDON::CServiceAddonManager &CServiceManager::GetServiceAddons()
+{
+  return *m_serviceAddons.get();
```

Comment:
This can be just `return *m_serviceAddons;`. Same as above (line 97) too.

Aside from that, this PR looks good.